docs/_data/nav.yml

```diff
@@ -6,9 +6,9 @@
   href: /support/
   category: support
 
-- title: Blog
-  href: /blog/
-  category: blog
+# - title: Blog
+#   href: /blog/
+#   category: blog
```

Comment:
like index.html for the docs does: https://github.com/facebook/nuclide/blob/master/docs/docs/index.html